src/client/game.cpp

```diff
@@ -4341,15 +4341,15 @@ void the_game(bool *kill,
 		}
 
 	} catch (SerializationError &e) {
-		error_message = std::string("A serialization error occurred:\n")
-				+ e.what() + "\n\nThe server is probably "
-				" running a different version of " PROJECT_NAME_C ".";
+		error_message = strgettext("A serialization error occurred:\n")
+				+ e.what() + gettext("\n\nThe server is probably "
+				" running a different version of ") + PROJECT_NAME_C + ".";
 		errorstream << error_message << std::endl;
 	} catch (ServerError &e) {
 		error_message = e.what();
 		errorstream << "ServerError: " << error_message << std::endl;
 	} catch (ModError &e) {
-		error_message = std::string("ModError: ") + e.what() +
+		error_message = strgettext("ModError: ") + e.what() +
```

Comment:
This needs to be left as-is since the mainmenu relies on the value.

Reply:
`N_` means don't translate the text, right?

Reply:
You can just leave it at `std::string(...` like before.